### PR TITLE
Add documentation outlining the new data format in v5

### PR DESCRIPTION
**Pull request by kevinlind**
## Description

Adds documentation outlining how the tracking data is mapped to the Experience Event's "data" element.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: Documentation/data-format.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Adobe Experience Platform Edge Bridge for iOS
+
+## Data Format
```
Comment (emdobrin) [inline Documentation/data-format.md:3]: how about simply Data format?
Also don't forget to add it to toc -> https://github.com/adobe/aepsdk-edgebridge-ios/blob/main/Documentation/README.md

File: Documentation/data-format.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+As of version 5.0.0 of AEP Edge Bridge for iOS, the following table lists the mapping of the `trackAction` and `trackState` parameters to the "data" element of the Experience Event sent to the Edge Network.
+
+
+| Data | Key in dispatched request | Comments |
+| --- | --- | ---
+| action | `data.__adobe.analytics.linkName` | Additionally, the field `data.__adobe.analytics.linkType` with value `lnk_o` is automatically included. |
+| state | `data.__adobe.analytics.pageName` | |
+| contextdata | `data.__adobe.analytics.contextData` | Context data is a map which includes the custom keys and values specified in the `trackAction` and `trackState` API calls. |
+| prefixed contextdata | `data.__adobe.analytics` | Context data keys prefixed with `&&` must be known to Analytics and are case sensitive. When mapped to the event, the key's name does not include the "&&" prefix.|
```
Comment (emdobrin) [inline Documentation/data-format.md:16]: Data column: `&& prefixed context data.`
Description column: For example &&products is sent as data.__adobe.analytics.products.

File: Documentation/data-format.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+## Data Format
+
+The AEP Edge Bridge extension maps the data sent in `trackAction` and `trackState` API calls to a specific format known to Analytics which requires no additional server-side mapping. This document outlines the data format applied to the tracking data.
+
+
+As of version 5.0.0 of AEP Edge Bridge for iOS, the following table lists the mapping of the `trackAction` and `trackState` parameters to the "data" element of the Experience Event sent to the Edge Network.
+
+
+| Data | Key in dispatched request | Comments |
```
Comment (emdobrin) [inline Documentation/data-format.md:11]: Key in dispatched request -> Key path in the network request
Comments -> Description

File: Documentation/data-format.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+| customer perspective | `data.__adobe.analytics.cp` | The customer perspective is automatically added to every tracking event. The values are either "foreground" or "background". |
+
+### Examples
+
+Given the track action call: 
+
+```swift
+MobileCore.track(action: "action name", data: ["key": "value", "&&products": ";Running Shoes;1;69.95;event1|event2=55.99;eVar1=12345"])
+```
+The resulting Experience Event has the following `data` element:
```
Comment (emdobrin) [inline Documentation/data-format.md:27]: data element may e confusing for Data Collection users. How about: The resulting Experience Event has the following payload:

File: Documentation/data-format.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
+# Adobe Experience Platform Edge Bridge for iOS
+
+## Data Format
+
+The AEP Edge Bridge extension maps the data sent in `trackAction` and `trackState` API calls to a specific format known to Analytics which requires no additional server-side mapping. This document outlines the data format applied to the tracking data.
+
+
+As of version 5.0.0 of AEP Edge Bridge for iOS, the following table lists the mapping of the `trackAction` and `trackState` parameters to the "data" element of the Experience Event sent to the Edge Network.
```
Comment (emdobrin) [inline Documentation/data-format.md:8]: "data" element -> "data" portion / node

**Review by stcheng-adobe — COMMENTED** [review]
Great job, I like the format.  I assume once the PR form Russ is finalized, we can add the link to all the prefixed key in the table.

**Comment by kevinlind** [comment]
> Great job, I like the format. I assume once the PR form Russ is finalized, we can add the link to all the prefixed key in the table.

Yes, we can include a link to the data-var-mapping.md file once his PR is merged and live.

**Review by emdobrin — APPROVED** [review]

**Pull request merged**